spec/integration/plugins/associates_spec.rb

```diff
@@ -58,33 +59,98 @@
     end
 
     context 'with a schema' do
-      include_context 'automatic FK setting', schema: true do
+      include_context 'automatic FK setting'
+
+      before do
+        configuration.relation_classes[1].class_eval do
+          schema do
+            attribute :id, ROM::SQL::Types::Serial
+            attribute :user_id, ROM::SQL::Types::ForeignKey(:users)
+            attribute :title, ROM::SQL::Types::String
+
+            associate do
+              belongs :users, as: :user
```

Comment:
Crossed my mind. It's gonna be super common to use a singular name so...what would be the downside if we did that? Can't think of anything.